### ISSUE TITLE
__global__ filters resulting in fatal error `*resources.xyz does not support custom properties`

### ISSUE THREAD

**Issue by zack-is-cool** [ekristen/aws-nuke#406]
I'm trying to filter globally using tags:

```yaml
accounts:
  "999999999999":
    filters:
      __global__:
        - property: "tag:Nuke"
          value: "false"
```

My assumption is that this would look for everything that has tags and exclude that, but it seems like this is filtering and throwing errors on resources that don't support properties.

This is from multiple different runs after excluding resources each time

```
*resources.BackupVaultAccessPolicy does not support custom properties
*resources.ResourceExplorer2Index does not support custom properties
*resources.DirectoryServiceDirectory does not support custom properties```

Do globals only work if you exclude anything that can't have properties? I'm not sure I'm configuring this properly.

**Comment by zack-is-cool** [comment]
I did just find this

https://github.com/rebuy-de/aws-nuke/issues/218#issuecomment-1387279644

**Comment by ekristen** [comment]
Are you sure it's exiting as fatal with those? It shouldn't be. Not all resources support the newer properties method, which lets you know the filter you have won't work. Not all resources support tags unfortunately.

**Comment by zack-is-cool** [comment]
The container I'm using appears to be using an old 3.28.0 which has that bug. I see..

okay. Yeah we can probably close this then 😬

**Comment by ekristen** [comment]
Ah yes, I broke 3.28.0 sorry about that, fixed in 3.29.0. I'm working to bring property support to everything and tags where supported.
